.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+# Changes
+
+<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
+
+<!-- Describe your changes here- ideally you can get that description straight from
+your descriptive commit message(s)! -->
+
+# Submitter Checklist
+
+See [the contribution guide](https://github.com/shipwright-io/build/blob/master/CONTRIBUTING.md)
```

Comment:
nit: /s/contribution/contributor/

Reply:
Shouldn't this also be in an HTML comment?

Reply:
switched to contributor in next push

Reply:
it was not a HTML comment in the the version in upstream Tekton I borrowed from @adambkaplan 

See https://raw.githubusercontent.com/tektoncd/triggers/master/.github/pull_request_template.md 
for example

Reply:
but if the majority is the break from that pattern and make this an HTML comment I'll defer and update accordingly

Reply:
Given the title is "Submitter checklist", I would expect there to be an actual checklist that contributors could mark off, such as:

```
- [ ] Includes tests if functionality changed/was added
- [ ] Includes docs if changes are user-facing
- [ ] Release notes block has been filled in, or marked NONE
```

Reply:
check list added @adambkaplan thanks